sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -568,7 +568,7 @@ public Builder(ConnectionPolicy connectionPolicy) {
                 this.bufferPageSize = DEFAULT_OPTIONS.bufferPageSize;
                 this.connectionAcquisitionTimeout = DEFAULT_OPTIONS.connectionAcquisitionTimeout;
                 this.connectTimeout = connectionPolicy.getConnectTimeout();
-                this.idleChannelTimeout = connectionPolicy.getIdleTcpConnectionTimeout();
+                this.idleChannelTimeout = connectionPolicy.getIdleConnectionTimeout();
```

Comment:
I think this is in-correct. Don't we want to use `getIdleTcpConnectionTimeout` ?

Reply:
I am pretty sure it will not compile - because we don't have this method any more :)